core/lib/spree/core/filters/date.rb

```diff
@@ -0,0 +1,8 @@
+module Spree
+  module Core
+    module Filters
+      class Date < Comparable
```

Comment:
https://github.com/rubocop-hq/ruby-style-guide#single-line-classes
What do you think?
I'm not a big fan of `Class.new(something)` but we can write it as `class Date < Comparable; end` to make it more compact

Reply:
I think it's good in situation where you have more code in module. In this case, it doesn't look too good in my opinion

```
module Spree
  module Core
    module Filters
      class Number < Comparable; end
    end
  end
end
```

We don't need compact code in this case imo